website/pages/en/quick-start.mdx

```diff
@@ -2,24 +2,20 @@
 title: Quick Start
 ---
 
-This guide will quickly take you through how to initialize, create, and deploy your subgraph to Subgraph Studio.
+This guide walks you through the basics of creating, deploying, publishing, and querying a subgraph on The Graph easily.
 
-Ensure that your subgraph will be indexing data from a [supported network](/developing/supported-networks).
-
-This guide is written assuming that you have:
+**Prerequisites for this guide:**
 
 - A crypto wallet
 - A smart contract address on the network of your choice
+- A subgraph that will be indexing data from a [supported network](/developing/supported-networks/)
```

Comment:
We are teaching them how to deploy a subgraph so they won't have a subgraph. I think at this stage we should also assume that they don't even know what Subgraph.

```suggestion
- A smart contract address on the [supported network](/developing/supported-networks/) of your choice
```